pyroute2/netlink/core.py

```diff
@@ -184,9 +184,9 @@ def __init__(
         self.local = threading.local()
         self.lock = threading.Lock()
         if use_event_loop:
-            self.local.event_loop = use_event_loop
             self.status['use_event_loop'] = True
             self.status['thread_id'] = id(threading.current_thread())
+            self.status['event_loop'] = id(use_event_loop)
```

Comment:
I do not understand this line, why are you calling id() on a boolean?

Reply:
Oh lol.

Reply:
Updated. Looks like status struct needs some typization. Give me some time to go through it tonight (but this version is already working)

Reply:
Shortly speaking, `status['use_event_loop']` might be either `False` or `True`. If it is `True` then we're using a predefined event loop, and must reject to work outside of the initial thread.

If it is `False`, then we try to get a running event loop from the current thread (`status['event_loop'] == 'auto'`), or create a new one (`status['event_loop'] == 'new'`).

Probably there might be a better way to express this choice.

Reply:
Yep, let me write it over with typization.